metron-platform/metron-writer/src/main/java/org/apache/metron/writer/BulkWriterComponent.java

```diff
@@ -118,12 +119,18 @@ public void commit(BulkWriterResponse response) {
 
   public void error(String sensorType, Throwable e, Iterable<Tuple> tuples, MessageGetStrategy messageGetStrategy) {
     LOG.error(format("Failing %d tuple(s); sensorType=%s", Iterables.size(tuples), sensorType), e);
-    MetronError error = new MetronError()
-            .withSensorType(Collections.singleton(sensorType))
-            .withErrorType(Constants.ErrorType.INDEXING_ERROR)
-            .withThrowable(e);
-    tuples.forEach(t -> error.addRawMessage(messageGetStrategy.get(t)));
-    handleError(tuples, error);
+    tuples.forEach(t -> {
+      MetronError error = new MetronError()
+              .withSensorType(Collections.singleton(sensorType))
+              .withErrorType(Constants.ErrorType.INDEXING_ERROR)
+              .withThrowable(e)
+              .addRawMessage(messageGetStrategy.get(t));
```

Comment:
While testing this, I am seeing a problem.  I am trying to decide whether we should call that a pre-existing condition and fix that problem after this PR goes in or whether the change here makes things any worse.  Let me know what you think.

I have described the problem in [METRON-1832](https://issues.apache.org/jira/browse/METRON-1832).  If any index destination goes down, error messages are continually recycled and grow larger and larger.  In addition, long sequence of escape characters are created.

Reply:
Thinking more about this some more... I feel that what I have described in METRON-1832 is a pre-existing condition.  I don't think this PR makes that pre-existing condition any worse.  This PR is a solid, necessary fix.  I am in favor of merging this and I will open a discuss thread on a resolution for METRON-1832.